frontend/packages/knative-plugin/src/components/overview/DeploymentOverviewList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { PodControllerOverviewItem } from '@console/shared';
+import { SidebarSectionHeading } from '@console/internal/components/utils';
+import { ResourceLink } from '@console/internal/components/utils';
+
+export type DeploymentOverviewListProps = {
+  current: PodControllerOverviewItem;
+};
+
+const DeploymentOverviewList: React.FC<DeploymentOverviewListProps> = ({ current }) => {
+  const namespace = current?.obj?.metadata?.namespace;
+  const deploymentData = _.get(current, 'obj.metadata.ownerReferences[0]', {});
```

Comment:
Any reason of not using optional chaining here? As you already have a check for deploymentData in line 17.  So, no need to pass empty object as default value.